iohk-monitoring/test/Cardano/BM/Test/Tracer.lhs

```diff
@@ -72,31 +72,31 @@ setupMockTrace (TraceConfiguration cfg mockSB name subTr) = do
 \end{code}
 
 \begin{code}
-renderNamedItemTracing' :: Show a => Tracer m String -> Tracer m (LogObject a)
-renderNamedItemTracing' = contramap $ \item ->
-    unpack (lo2name item) ++ ": " ++ show (loContent item) ++ ", (meta): " ++ show (loMeta item)
+renderNamedItemTracing' :: Show a => Tracer m String -> Trace m a
+renderNamedItemTracing' = contramap $ \(ctx,item) ->
```

Comment:
Same as above.

Reply:
form its usage further down it uses 'appendName' to enter a name context and then render it to a string for output on stdout.